heroic-dist/src/test/java/com/spotify/heroic/analytics/bigtable/HeroicMetricsConfigurationTest.java

```diff
@@ -107,55 +110,57 @@ public void testBigtableLimitsConfig() throws Exception {
         // Check that the BigTableBackend's maxWriteBatchSize was picked up
         // from the heroic-all.yml config file
         instance.inject(
-            coreComponent -> {
-                var metricManager =
-                    (LocalMetricManager) ((DaggerCoreComponent) coreComponent).metricManager();
-                var analyticsBackend =
-                    metricManager
-                        .groupSet()
-                        .useGroup("bigtable")
-                        .getMembers()
-                        .toArray(new BigtableAnalyticsMetricBackend[0])[0];
-                var bigtableBackend = (BigtableBackend) analyticsBackend.getBackend();
+                coreComponent -> {
+                    var metricManager =
+                            (LocalMetricManager) ((DaggerCoreComponent) coreComponent)
+                                    .metricManager();
+                    var analyticsBackend =
+                            metricManager
+                                    .groupSet()
+                                    .useGroup("bigtable")
+                                    .getMembers()
+                                    .toArray(new BigtableAnalyticsMetricBackend[0])[0];
+                    var bigtableBackend = (BigtableBackend) analyticsBackend.getBackend();
 
                     assertEquals(EXPECTED_MAX_WRITE_BATCH_SIZE,
-                            bigtableBackend.getMaxWriteBatchSize());
+                            (long)bigtableBackend.metricsConnectionSettings().getMaxWriteBatchSize());
```

Comment:
Why are the `long` conversions needed? All of the EXPECTED_ variables are already ints, aren't they?